components/camel-mina2/src/main/java/org/apache/camel/component/mina2/Mina2Producer.java

```diff
@@ -65,6 +65,7 @@
 public class Mina2Producer extends DefaultProducer implements ServicePoolAware {
 
     private static final Logger LOG = LoggerFactory.getLogger(Mina2Producer.class);
+    private final ResponseHandler handler;
     private IoSession session;
     private CountDownLatch latch;
```

Comment:
Thanks for the reviews! I will provide a new patch.
Op 18 jun. 2016 2:26 p.m. schreef "Gijsbert van den Brink" <
notifications@github.com>:

> In
> components/camel-mina2/src/main/java/org/apache/camel/component/mina2/Mina2Producer.java
> <https://github.com/apache/camel/pull/1043#discussion_r67600299>:
>
> > @@ -65,6 +65,7 @@
> >  public class Mina2Producer extends DefaultProducer implements ServicePoolAware {
> >
> >      private static final Logger LOG = LoggerFactory.getLogger(Mina2Producer.class);
> > +    private final ResponseHandler handler;
> >      private IoSession session;
> >      private CountDownLatch latch;
>
> Maybe rename this to responseLatch (or something)? To avoid confusion
> about what the latches are used for.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/apache/camel/pull/1043/files/fe41b1bb9ac743214a6f6baba393a251e55037fc#r67600299>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AFC1lYinKwwwmI95aoVyKEQEpkdHyZuyks5qM-QAgaJpZM4I4bps>
> .
>